packages/analytics/src/integrations/Omnitracking/omnitracking-helper.ts

```diff
@@ -323,6 +323,27 @@ export const getValParameterForEvent = (
   return JSON.stringify(valParameters);
 };
 
+/**
+ * Get the product unit sale price from a specific product.
+ *
+ * @param productData - Product data.
+ *
+ * @returns The unit sale price value.
```

Comment:
Check the changes requested for the equivalent PR for v1 and apply them here, please.

Reply:
Done 👍🏼